docs/doc/30-reference/30-sql/00-ddl/90-share/02-drop-share.md

```diff
@@ -0,0 +1,21 @@
+---
+title: DROP SHARE
+---
+
+Deletes a share. When you delete a share, the organizations added to the share will no longer have access to the database created from the share.
+
+For more information about how to work with shares in Databend Cloud, see [SHARE](index.md).
```

Comment:
We should remove this line for all.
Databend Cloud Share will be a UI operation, not a SQL command.
This share doc is mainly for databend with SQL commands.